### PR TITLE
Enable deployment of exhibitions from local source

### PR DESCRIPTION
**Pull request by AudreyAltman**
This addresses ticket #7822 https://issues.dp.la/issues/7822

### CODE REVIEW

**Comment by AudreyAltman** [comment]
I tested this locally by deploying a local branch of the exhibitions app, and it seems to be working as expected.

**Comment by anarchivist** [comment]
I'm :+1:, but maybe have @markbreedlove take a look too before we merge?

File: ansible/roles/omeka/files/copy_local_app.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  echo "Local directory /exhibitions_dev does not exist" >&2
+  exit 1
+fi
+
+if [ ! -d /home/dpla/exhibitions-local ]; then
+  mkdir /home/dpla/exhibitions-local
+fi
+
+rsync -rptl --delete --checksum \
+    --exclude 'application/logs' \
```
Comment (markbreedlove) [inline ansible/roles/omeka/files/copy_local_app.sh]: Can you confirm that `application/logs` gets created by Omeka when it's started?  In other roles, like `api`, there's a build script that ensures that such a directory exists in the destination, e.g. `/srv/www/api`, when the "build" script is run to copy the app from the home directory.  If it got created in your dev environment, then this should be OK.

**Comment by markbreedlove** [comment]
@AudreyAltman, just fixed something in https://github.com/dpla/automation/commit/3d9eb56688e1677c8b83996c9e1763a4c0a63a47 related to my last comment, and related to the rsync error that you reported to me before.

**Comment by anarchivist** [comment]
I'm :+1: if @markbreedlove's good on this. Shall we merge?

**Comment by markbreedlove** [comment]
I'm fine with merging it.  @AudreyAltman, would you like to squash it first?

File: ansible/roles/omeka/files/copy_local_app.sh (hunk 1 above)
Comment (AudreyAltman) [inline ansible/roles/omeka/files/copy_local_app.sh]: Yes, I can confirm that `application/logs` was created by Omeka.

**Comment by AudreyAltman** [comment]
:squash:ed and ready to :merge: @markbreedlove or @anarchivist

**Pull request merged**